shell/platform/linux/fl_view.cc

```diff
@@ -348,21 +349,19 @@ static gboolean fl_view_motion_notify_event(GtkWidget* widget,
 static gboolean fl_view_key_press_event(GtkWidget* widget, GdkEventKey* event) {
   FlView* self = FL_VIEW(widget);
 
-  fl_key_event_plugin_send_key_event(self->key_event_plugin, event);
-  fl_text_input_plugin_filter_keypress(self->text_input_plugin, event);
-
-  return TRUE;
+  return fl_key_event_plugin_send_key_event(self->key_event_plugin, event)
+             ? TRUE
```

Comment:
I think the bool -> gboolean mapping shouldn't require the `? TRUE : FALSE`.

Reply:
Removed.